modules/Bio/EnsEMBL/Compara/PipeConfig/UpdateMemberNamesDescriptions_conf.pm

```diff
@@ -53,6 +53,7 @@ sub default_options {
 
         #Pipeline capacities:
         'update_capacity'                           => '5',
+        'exclude_species'  => [],
```

Comment:
```suggestion
        #Outgroup species to exclude from GenomeDBFactory
        'exclude_species'  => [],
```

Reply:
Can we stick a comment or a new line in above the parameter, just to separate it from the pipeline capacities, because it is not a pipeline capacity?